satsense/generators.py

```diff
@@ -61,17 +92,36 @@ def load_image(self, itype, windows):
         self._padding = tuple(
             max(math.ceil(0.5 * w[i]) for w in windows) for i in range(2))
 
+        block = self.get_blocks()
+        image = self.image.copy_block(block)
+        self._image_cache = image[itype]
+        self.loaded_itype = itype
+
+    def get_blocks(self):
```

Comment:
```suggestion
    def _get_blocks(self):
```
This should probably be a private method